hedera-node/src/main/java/com/hedera/services/txns/file/FileSysDelTransitionLogic.java

```diff
@@ -68,50 +65,29 @@ public FileSysDelTransitionLogic(
 
 	@Override
 	public void doStateTransition() {
+		/* --- Extract from gRPC --- */
 		var op = txnCtx.accessor().getTxn().getSystemDelete();
```

Comment:
I think that we should declare local variables as final where possible, including - when refactoring legacy code, including - in the constructors' parameters.